service.yml

```diff
@@ -1,5 +1,10 @@
-director: Delaney Manders
```

Comment:
Why are we removing this? Also, it should've been a github account linked with the director

Reply:
It's just not needed. I think director ownership is deprecated anyhow.